cmp/feedback/FeedbackDialog.js

```diff
@@ -12,7 +12,7 @@ import {textAreaField, toolbar} from 'hoist/cmp';
 
 /**
  * A simple dialog component to collect user feedback from directly within the application.
- * @see FeedbackService
+ * See FeedbackService
```

Comment:
No `@see`?  I was finding that IntelliJ was typically able to figure these out and provide the ctrl+click option, so I was kind of happy to use them, especially  when we had a case like this one where that's all we planned to say on the line.

Reply:
Agree its useful, and cool even if it doesn't link in tools.

Reply:
OK. I was actually using them in a few places as well, but thought I was drinking too much kool-Aid. (Was using with @link as well is THAT too much? 

@see {@link http://usejsdoc.org/tags-see.html}